docs/protocol.html

```diff
@@ -200,7 +200,7 @@ <h5 class="anchor-heading"><a id="protocol_error_codes" class="anchor-link"></a>
 <!--#include virtual="generated/protocol_errors.html" -->
 
 <h5 class="anchor-heading"><a id="protocol_api_keys" class="anchor-link"></a><a href="#protocol_api_keys">Api Keys</a></h5>
-<p>The following are the numeric codes that the ApiKey in the request can take for each of the below request types.</p>
+<p>The following are the numeric codes that the ApiKey which last version is stable in the request can take for each of the below request types.</p>
```

Comment:
maybe `stable ApiKey` is good enough?

Reply:
Yes, If ApiKey only have last version, I agree that use `stable ApiKey` is good enough